### DISCUSSION TITLE
How does simnibs4 generate individual specific input currents to produce maximum electric field strength?

### DISCUSSION THREAD

**Discussion by Jaycob-jh** [simnibs/simnibs#332] [9. Results and analysis]
Dear all,
How can I use simnibs4 to get the input current for each specific electrode?
I have now simulated and obtained the results using a custom current input and electrode configuration scheme. However, I would like to further use simnibs4 to help me simulate how much current should be input to each electrode to enable the individual to generate the maximum electric field strength in this area. How does this work?

**Comment by TorgeW** [comment]
It seams like you want to optimize your tdcs setup, have a look at 
https://simnibs.github.io/simnibs/build/html/tutorial/optimization.html

If you still have questions feel free to ask! ! :)
